app/helpers/app_config_helper.rb

```diff
@@ -0,0 +1,7 @@
+module AppConfigHelper
+  def get_app_config(key)
+    app_config = AppConfig.where(key: key).first
```

Comment:
Shouldn't we just make 'key' unique?

Reply:
It still feels not great for this particular flag to add another db call on each request, but we can re-evaluate 😬